src/contracts/core/DelegationManager.sol

```diff
@@ -664,6 +706,7 @@ contract DelegationManager is
         IStrategy[] memory strategies,
         uint256[] memory shares
     ) internal returns (bytes32) {
+        require(!isCompletableHandoff(operator), "DelegationManager._removeSharesAndQueueWithdrawal: staker's operator is in the middle of a handoff");
```

Comment:
so stakers cannot queue new withdrawals once the handoff is completable? 
this is enforcing that shares must be moved to the AllocatorManager and a new withdrawal queued there instead, right?